Sources/Purchasing/ProductsManager.swift

```diff
@@ -119,9 +121,11 @@ private extension ProductsManager {
             productsFetcherSK1.products(withIdentifiers: removedProductIdentifiers, completion: { result in
                 switch result {
                 case.success:
```

Comment:
Hun looks like this is missing a space?
```suggestion
                case .success:
```